jobs/build/appregistry/Jenkinsfile

```diff
@@ -4,14 +4,11 @@ def processImages(lines) {
     lines.split().each { line ->
         // label, name, nvr, version
         def fields = line.split(',')
-        if (fields[0] == 'true') {
```

Comment:
I removed this filter for `appregistry=true` because so far we have been passing only operators to this job.
Is there any chance we pass other images (non operators) ?

Reply:
This job runs against all images normally, only the ones with this label are put through the operator process. So we still need the filter.
However it should be looking for the label to exist at all now, not to be `true`. It won't be `true`.

Reply:
:heavy_check_mark: